frontend/src/components/student/AllActivities/CombatTask/CombatTask.js

```diff
@@ -91,33 +93,48 @@ export default function CombatTask() {
             </HeaderCol>
             <div>
               <h5>{task.description}</h5>
+              {task.points !== null && (
+                <>
+                  <SmallDivider />
+                  <p>
+                    Zdobyte punkty: <strong>{task.points}</strong>
+                  </p>
+                  {task.remarks && (
+                    <>
+                      <p>Uwagi od prowadzącego:</p>
+                      <p>{task.remarks}</p>
+                      {task.feedbackFile && <FeedbackFileService feedbackFile={task.feedbackFile} />}
+                    </>
+                  )}
```

Comment:
Mimo wszystko dałbym widoczność tekstu "Uwagi od prowadzącego" na stałe i w zależności od istnienia remarks wstawił bym albo te remarks albo tekst typu "Brak uwag" lub po prostu "-", żeby student miał świadomość, że prowadzący nie dał uwag. Gdybyś odpalił to po raz pierwszy to pewnie byś nie ogarnął, że prowadzący nie dodał uwag a np sam plik lub tylko punkty. Ja bym przynajmniej nie ogarnął